### DISCUSSION TITLE
goctl 生成的代码如何让多个handler在同一个文件下呢

### DISCUSSION THREAD

**Discussion by hydrogenwuyan** [zeromicro/go-zero#269] [Q&A] [answered]
handler文件太多，一个功能的增删改查分成多个文件没必要。

**Comment by kevwan** [comment, marked as answer]
api文件里可以加group指令控制

**Reply by kevwan** [reply]
还有goctl-go-compact可以用

**Reply by linden-in-China** [reply]
这个group指令，是把不同的handler文件放在同一个目录吧。效果和goctl-go-compact不一样。可不可以通过api文件的配置，直接就生成goctl-go-compact的效果？
